samples/test/util.py

```diff
@@ -264,8 +265,9 @@ def _create_run():
             host=metadata_service_host,
             port=metadata_service_port,
         )
-        callback(run_pipeline=run_pipeline,
-                 mlmd_connection_config=mlmd_connection_config)
+        callback(
```

Comment:
it seems that file formatting has changed, how do I upgrade to use the new format?

Reply:
I was using the same style as https://github.com/kubeflow/pipelines/blob/master/sdk/.style.yapf in my local default config.